Week3/homework/project-card-validator.js

```diff
@@ -0,0 +1,51 @@
+// Checks the lenght of the string, returns true if it is 16.
+function checkLength(item){
+  return item.length === 16;
+}
+
+// Checks if the string consists of only digits.
+function checkAllDigit(item){
+  return +item == item;
+}
+
+// Checks if all the chars are same.
+function checkAllSame(item){
+  return item % 1111111111111111 === 0;
```

Comment:
Aahaan. This is an interesting way of checking. I would also like to introduce you to a new topic in JavaScript called Sets(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). You can perhaps use it as well to achieve the desired behaviour.

Reply:
Himm, something like this maybe???
return new Set(item.split('')).size > 1